dev-libs/libbpf/libbpf-9999.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs git-r3
```

Comment:
Generally in packages with a live ebuild, for ease of maintenance I prefer to do something like:
```ebuild
inherit toolchain-funcs
if [[ ${PV} =~ [9]{4,} ]]; then
	inherit git-r3
	EGIT_REPO_URI="https://github.com/libbpf/libbpf.git"
else
	SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
	KEYWORDS="~amd64 ~arm64 ~x86"
fi
```
That way the 9999 can often be identical to the latest version of the ebuild, barring patches.